.github/workflows/build-binaries.yml

```diff
@@ -0,0 +1,100 @@
+name: Build LLVM binaries
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # every month to regenerate ccache
```

Comment:
@antonbaliasnikov just curious, why do we need this?

Reply:
This is to properly regenerate `ccache` that can be used across LLVM builds in PRs.

For front-end compilers, it's easy to understand when a new cache should be regenerated - each time an LLVM version in the `LLVM.lock` file changes, but for LLVM backend ideally we need to regenerate the cache each time a new commit is pushed to `main`, but it's too expensive - we'll be out of GitHub cache space pretty soon. So, here I'm introducing a monthly trigger that will update the `main` cache each month or by request.

Each month is good to keep the cache in almost 98-99% hits and at the same time not overload GitHub cache.